vaadin-dialog-flow-parent/vaadin-dialog-flow/src/main/java/com/vaadin/flow/component/dialog/Dialog.java

```diff
@@ -917,6 +919,45 @@ protected void onAttach(AttachEvent attachEvent) {
         initHeaderFooterRenderer();
     }
 
+    /**
+     * <p>
+     * Description copied from corresponding location in WebComponent:
```

Comment:
Let's clean up the generated stuff that is not helpful for Flow users like `Description copied from corresponding location in WebComponent:`